pkg/sqlcache/db/client.go

```diff
@@ -40,6 +41,8 @@ const (
 
 	informerObjectCachePerms fs.FileMode = 0o600
 
+	maxBeginTXAttemptsOnBusyErrors = 3
```

Comment:
Should we expose this value via config?

Reply:
Hmm I have mixed feelings, it's probably something we will never need to tweak.
The error we are trying to mitigate is very specific and unlikely to happen, but still possible, and the only way to deal with it is to retry.
If it happens more than 3 times in a row it may be a sign of a bigger problem and we may not want to allow to ignore that?